CONTRIBUTING.md

```diff
@@ -1,21 +1,23 @@
 
-Delta Lake is supported by a wide set of developers from over 50 organizations across multiple repositories.  Since 2019, more than 190 developers have contributed to Delta Lake!  The Delta Lake community is growing by leaps and bounds with more than 5500 members in the [Delta Users slack](https://dbricks.co/delta-users-slack)).
+Delta Lake is supported by a wide set of developers from over 50 organizations across multiple repositories.  Since 2019, more than 190 developers have contributed to Delta Lake!  The Delta Lake community is growing by leaps and bounds with more than 6000 members in the [Delta Users slack](https://dbricks.co/delta-users-slack)).
 
 For more information, please refer to the [founding technical charter](https://delta.io/pdfs/delta-charter.pdf).
 
 # Communication
-- Before starting work on a major feature, please reach out to us via [GitHub](https://github.com/delta-io/delta/issues), [Slack](https://dbricks.co/delta-users-slack), [email](https://groups.google.com/g/delta-users), etc. We will make sure no one else is already working on it and ask you to open a GitHub issue.
+- Before starting work on a major feature, please reach out to us via [GitHub](https://github.com/delta-io/delta/issues), [Slack](https://delta-users.slack.com/join/shared_invite/zt-165gcm2g7-0Sc57w7dX0FbfilR9EPwVQ#/shared-invite/email), [email](https://groups.google.com/g/delta-users), etc. We will make sure no one else is already working on it and ask you to open a GitHub issue.
 - A "major feature" is defined as any change that is > 100 LOC altered (not including tests), or changes any user-facing behavior.
 - We will use the GitHub issue to discuss the feature and come to agreement.
 - This is to prevent your time being wasted, as well as ours.
 - The GitHub review process for major features is also important so that organizations with commit access can come to agreement on design.
 - If it is appropriate to write a design document, the document must be hosted either in the GitHub tracking issue, or linked to from the issue and hosted in a world-readable location. Examples of design documents include [sample 1](https://docs.google.com/document/d/16S7xoAmXpSax7W1OWYYHo5nZ71t5NvrQ-F79pZF6yb8), [sample 2](https://docs.google.com/document/d/1MJhmW_H7doGWY2oty-I78vciziPzBy_nzuuB-Wv5XQ8), and [sample 3](https://docs.google.com/document/d/19CU4eJuBXOwW7FC58uSqyCbcLTsgvQ5P1zoPOPgUSpI).
 - Specifically, if the goal is to add a new extension, please read the extension policy.
-- Small patches and bug fixes don't need prior communication.
+- Small patches and bug fixes don't need prior communication. If you identified a bug and have ways to solve it, please create an [issue](https://github.com/delta-io/delta/issues).
```

Comment:
if they have ways to solve the issue, shouldn't they create a PR instead of a github issue?